cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js

```diff
@@ -41,7 +42,82 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
         }
       });
     });
+    describe('Most Watched component', () => {
+      before(() => {
+        cy.getToggles(config[service].name);
+      });
+      // Allows constructing most watched path with variant
+      const serviceVariant = variant === 'default' ? '' : `/${variant}`;
+
+      const mostWatchedPath = `/${config[service].name}/mostwatched${serviceVariant}.json`;
+      it('should show/not show the Most Watched component if the toggle is enabled for the service', function test() {
+        // Find if the service has component enabled in toggles
+        cy.fixture(`toggles/${config[service].name}.json`).then(toggles => {
+          const mostWatchedIsEnabled = path(
+            ['mostPopularMedia', 'enabled'],
+            toggles,
+          );
+          cy.log(`Service toggle enabled ${mostWatchedIsEnabled}`);
+          // If toggle is enabled for that service, also check there are more than 0 records to show
+          if (mostWatchedIsEnabled) {
+            cy.request(mostWatchedPath).then(({ body: mostWatchedJson }) => {
+              if (mostWatchedJson.totalRecords > 0) {
+                cy.log('Enough records to show component');
+                cy.get('[data-e2e=most-watched-heading]').should('exist');
+              } else {
+                cy.log('Not enough records to show component');
+
+                cy.get('[data-e2e=most-watched-heading]').should('not.exist');
+              }
+            });
+          } else {
+            cy.get('[data-e2e=most-watched-heading]').should('not.exist');
+          }
+        });
+      });
+      it('Most Watched component shows correct number of items', function test() {
```

Comment:
```suggestion
      it('should show correct number of items', function test() {
```

('component' is referenced when the initial 'describe' is declared)